CalcualtorLogic.js

```diff
@@ -113,6 +113,7 @@ class Calculator {
     }
     onOperatorButtonClick(event) {
         const curntRsltAndOperclkdBtn = event.target.dataset.operator;
+        operator = curntRsltAndOperclkdBtn;
```

Comment:
you can't access the `operator` property like this
you need to do `this.operator` every time you want to access its value inside methods

Reply:
just I pushed this update before reading your beautiful comment
and noticed that this property doing the job of variables in the previous implantation of the calculator
Thanks !